### ISSUE TITLE
GenericChart <a> Broken Link for Margin

### ISSUE THREAD

**Issue by Daniel-Peladeau** [Nicholas-Nazario/twitterLens#67]
./src/components/GenericChart.js
  Line 12:21:  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid

**Comment by Daniel-Peladeau** [comment]
Fixed this issue, we are using <a> to do styling for the margins here. This does not need a link, so an empty link of "# " is the preferred standard.
